### PR TITLE
Shh ignore manifestjson updates

### LINKED ISSUE #59: Ignore changes to MANIFEST.json in nonini_wpt_meta
The rule as written is too broad and has false positives like https://github.com/servo/servo/pull/9632 .

### PR DESCRIPTION
**Pull request by g-k**
Fixes #59

and mozilla-sync.

### CODE REVIEW

File: handlers/nonini_wpt_meta/__init__.py [hunk 1]
```diff
@@ -10,8 +10,15 @@ class NonINIWPTMetaFileHandler(EventHandler):
         'tests/wpt/mozilla/meta',
     )
 
+    FALSE_POSITIVE_SUBSTRINGS = (
+        '.ini',
```
Comment (KiChjang) [inline handlers/nonini_wpt_meta/__init__.py]: Why are .ini files considered false positives?

Comment (jdm) [inline handlers/nonini_wpt_meta/__init__.py]: Because we don't want to warn about them.

**Comment by jdm** [comment]
Looks great @g-k! Please squash the commits together and I'll merge!

File: handlers/nonini_wpt_meta/__init__.py (hunk 1 above)
Comment (KiChjang) [inline handlers/nonini_wpt_meta/__init__.py]: Oh right, my logic is messed up -_-

**Comment by jdm** [comment]
Thanks!

**Pull request merged**